apps/api/src/app/change/dtos/change-request.dto.ts

```diff
@@ -2,6 +2,7 @@ import { ApiProperty } from '@nestjs/swagger';
 import { IsDefined, IsString } from 'class-validator';
 import { PaginationRequestDto } from '../../shared/dtos/pagination-request';
 
+// TODO: change promoted type to boolean
```

Comment:
I think we could either implement the todos or create tasks for them. if we will create the tasks it will be easier to prioritize them, then having them as todos in the code. wdyt?

Reply:
Sure I will create linear ticket

Reply:
https://linear.app/novu/issue/NV-2597

Reply:
remove all comments